vhdbuilder/packer/install-dependencies.sh

```diff
@@ -446,7 +446,7 @@ wait ${image_pids[@]}
 
 watcher=$(jq '.ContainerImages[] | select(.downloadURL | contains("aks-node-ca-watcher"))' $COMPONENTS_FILEPATH)
 watcherBaseImg=$(echo $watcher | jq -r .downloadURL)
-watcherVersion=$(echo $watcher | jq -r .multiArchVersions[0])
+watcherVersion=$(echo $watcher | jq -r .multiArchVersionsV2)
```

Comment:
don't we need to also index with 0?

Reply:
yeah we do: https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/prefetch/internal/containerimage/testdata/components.json#L486-L493